.github/main.yml

```diff
@@ -19,10 +19,16 @@ jobs:
         uses: subosito/flutter-action@v2
         with:
           flutter-version: 3.19.5
+      
+      - name: Check Formatting
+        run: dart format --set-exit-if-changed lib test
```

Comment:
The formatting command won't be able to run until the dependencies command has run. So best to add this after the `Get dependencies` step below.